ImageSaver/app/src/main/res/layout/fragment_image_query.xml

```diff
@@ -0,0 +1,65 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" tools:context="com.example.jong.imagesaver.Image_query">
+
+    <!-- TODO: Update blank fragment layout -->
+
+
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/save_image"
+        android:textSize="@dimen/button_size"
+        android:id="@+id/save_button"
+        android:layout_gravity="center_horizontal|bottom" />
+
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/previous"
+        android:textSize="@dimen/button_size"
+        android:id="@+id/prev_image"
+        android:layout_marginLeft="10dp"
+        android:layout_gravity="start|bottom" />
+
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/next"
+        android:textSize="@dimen/button_size"
+        android:id="@+id/next_image"
+        android:layout_marginRight="10dp"
+        android:layout_gravity="end|bottom" />
+
```

Comment:
make a style for components used more than once (like these buttons) that have similar features